capellambse/diagram/__init__.py

```diff
@@ -13,3 +13,17 @@
 from ._diagram import *
 from ._json_enc import *
 from ._styleclass import *
+
+import typing as t
+
+if not t.TYPE_CHECKING:
+    from ._vector2d import __all__ as _all1
+    from .capstyle import __all__ as _all2
+    from ._diagram import __all__ as _all3
+    from ._json_enc import __all__ as _all4
+    from ._styleclass import __all__ as _all5
+
+    __all__ = [*_all1, *_all2, *_all3, *_all4, *_all5]
+
+    del _all1, _all2, _all3, _all4, _all5
+del t
```

Comment:
We need to assemble the `__all__` list in order for Sphinx to detect what's being exported here. If not for this block, the auto-generated documentation page would be (almost) empty, and not show any of the exported classes.

I'm not entirely sure why this is necessary myself. Presumably because the actual class definitions are in (private) modules and therefore invisible by default.